crates/core_arch/src/arm/neon/mod.rs

```diff
@@ -3736,6 +3736,11 @@ pub unsafe fn vget_lane_u8<const IMM5: i32>(v: uint8x8_t) -> u8 {
     simd_extract(v, IMM5 as u32)
 }
 
+/// Duplicate vector element to vector or scalar.
+/// This instruction duplicates the vector element at the specified element index
+/// in the source SIMD&FP register into a scalar or each element in a vector,
+/// and writes the result to the destination SIMD&FP register.
```

Comment:
The doc comment should be duplicated for each function so that it appears properly in rustdoc.

Reply:
Done. Thanks. Delete redundant comments to keep consistent with the previous concise comment style.

Reply:
> The doc comment should be duplicated for each function so that it appears properly in rustdoc.

Done. Thanks. 
Remove redundant comments, consistent with the previous concise comment style